log_drains.go

```diff
@@ -15,6 +17,15 @@ type LogDrain struct {
 	URL   string `json:"url"`
 }
 
+type errorRes struct {
+	Error string `json:"error"`
+}
+
+type LogDrainRes struct {
+	LogDrain LogDrain `json:"drain"`
```

Comment:
I think I prefer the attribute name to be the same as in the JSON output:
```suggestion
	Drain LogDrain `json:"drain"`
```